app/views/layouts/main.html

```diff
@@ -1,6 +1,21 @@
+<!-- prettier-ignore -->
+
 {#
 For guidance on how to use layouts see:
 https://prototype-kit.service.gov.uk/docs/how-to-use-layouts
 #}
 
 {% extends "govuk-prototype-kit/layouts/govuk-branded.njk" %}
+{% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
+{% from "govuk/components/header/macro.njk" import govukHeader %}
+
+
+{% block header %} {{ govukHeader({ homepageUrl: "/" }) }}
+<link rel="stylesheet" href="/public/css/styles.css" />
+
+<div class="govuk-width-container">
+  {{ govukPhaseBanner({ tag: { text: "Alpha" }, html: 'This is a new service –
```

Comment:
did we want to change the text of the banner to not include the word "service"?

Reply:
I think so, yes, just waiting to hear back on what we should show instead

Reply:
The text has been removed entirely for now, I'll create a ticket which we can bring in once we know what the text should be